client-rpc/src/rpc/multisig_rpc.rs

```diff
@@ -5,12 +5,19 @@ use jsonrpc_derive::rpc;
 use chain_core::common::{H256, HASH_SIZE_256};
 use chain_core::tx::data::Tx;
 use client_common::{Error, ErrorKind, PublicKey, Result as CommonResult, ResultExt, SecKey};
+use client_core::types::AddressType;
 use client_core::{MultiSigWalletClient, WalletClient};
 
 use crate::server::{to_rpc_error, WalletRequest};
 
 #[rpc]
 pub trait MultiSigRpc: Send + Sync {
+    #[rpc(name = "multiSig_newAddressPublicKey")]
+    fn new_address_public_key(&self, request: WalletRequest) -> Result<String>;
+
+    #[rpc(name = "multiSig_listAddressPublicKeys")]
```

Comment:
The RPC endpoint name change is breaking, the `test_watch_only_wallet.py` and `chainrpc.py` needs to change accordingly.